### PR TITLE
Publish Content - Updates per Jesus

### PR DESCRIPTION
**Pull request by paigehargrave**
Please check the syntax of the links here and the wording style - thank you!

### CODE REVIEW

File: modules/ROOT/pages/publish-content.adoc [hunk 1]
```diff
@@ -6,6 +6,16 @@ endif::[]
...
 With ACM you can create, organize, and publish content articles including news, tutorials, developer blog posts, and events.
 
+You can author ACM content in either of the following ways:
+
+. In https://help.salesforce.com/articleView?id=community_managed_content_overview.htm&type=5[SalesforceCMS].
+. In https://help.salesforce.com/articleView?id=communities_cms_connect.htm&type=5[an external CMS system through 
+CMS Connect].
+
+To display content in community pages, you must organize content in COLLECTIONS using the *Manage CMS content* section 
+of the ACM admin interface, as shown in https://drive.google.com/open?id=1ZxQ2lvivxnXDKB8qfJX3qZM-Szrzx4Dl[this
```
Comment (jdeoliveira) [inline modules/ROOT/pages/publish-content.adoc]: we're not planning to release these videos externally - we'd need to explain the video here in prose format ;)

File: modules/ROOT/pages/publish-content.adoc [hunk 2]
```diff
@@ -6,6 +6,16 @@ endif::[]
...
+. In https://help.salesforce.com/articleView?id=communities_cms_connect.htm&type=5[an external CMS system through 
+CMS Connect].
+
+To display content in community pages, you must organize content in COLLECTIONS using the *Manage CMS content* section 
+of the ACM admin interface, as shown in https://drive.google.com/open?id=1ZxQ2lvivxnXDKB8qfJX3qZM-Szrzx4Dl[this 
+video with a similar talk track].
+
 == Create a New Article
 
 In the ACM administrator interface, select *Manage CMS Content*.
```
Comment (jdeoliveira) [inline modules/ROOT/pages/publish-content.adoc]: we should describe here the happy path to create an article in salesforce CMS:
- in ACM admin, select manage CMS content
- click on Salesforce CMS to go to Salesforce CMS component
- Open the workspace for the community you're working with
- see list of existing articles - click new to create a new one
- enter title, excerpt, image, and body
- click save draft and keep editing or click PUBLISH to make it live (if it is in a published collection as explained below on the collections section)

File: modules/ROOT/pages/publish-content.adoc [hunk 3]
```diff
@@ -6,6 +6,16 @@ endif::[]
...
+
+. In https://help.salesforce.com/articleView?id=community_managed_content_overview.htm&type=5[SalesforceCMS].
+. In https://help.salesforce.com/articleView?id=communities_cms_connect.htm&type=5[an external CMS system through 
+CMS Connect].
+
+To display content in community pages, you must organize content in COLLECTIONS using the *Manage CMS content* section 
+of the ACM admin interface, as shown in https://drive.google.com/open?id=1ZxQ2lvivxnXDKB8qfJX3qZM-Szrzx4Dl[this 
+video with a similar talk track].
+
 == Create a New Article
```
Comment (jdeoliveira) [inline modules/ROOT/pages/publish-content.adoc]: this should be called Create a new Article in SalesforceCMS

**Review by jdeoliveira — DISMISSED** [review]
Please make the two updates I've commented in-line - thank you!

**Comment by paigehargrave** [comment]
@jdeoliveira - I've restructured and updated this - PTAL - thank you!

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**